game/tests.py

```
@@ -407,6 +407,57 @@ def test_client_ip_uses_rightmost_untrusted_hop(self):
+            follow=True,
+        )
+        self.assertRedirects(response, self.done_url)
+        self.assertEqual(len(mail.outbox), 1)
+        self.assertNotContains(response, 'Select Your Account')
+
+    def test_multi_account_email_submission(self):
+        # Create a second user with the same email
+        User.objects.create_user(
+            username='resetplayer2',
+            email='reset@example.com',
+            password='StrongPass1234!',
```

Ruff (0.15.20)

[error] Possible hardcoded password assigned to argument: "password"

(S106)